packages/traceloop-sdk/traceloop/sdk/__init__.py

```diff
@@ -47,6 +48,7 @@ def init(
         propagator: TextMapPropagator = None,
         traceloop_sync_enabled: bool = True,
         resource_attributes: dict = {},
+        instruments=None
```

Comment:
Add type to the argument

Reply:
not sure about this one, should it be something like `array = None or []`? sorry if it's a dumb question

Reply:
No worries :) 
I mean `instruments: Set[Instruments] = {}`